src/status_im/contexts/wallet/create_account/edit_derivation_path/view.cljs

```diff
@@ -1,51 +1,66 @@
     | -> (clear-action) -> empty -> (derive-action) -> choose -> (choose-action) -> show"
-  [{:keys [on-reset on-reveal]}]
+  [{:keys [on-reset]}]
   (let [top                (safe-area/get-top)
         bottom             (safe-area/get-bottom)
-        state              (reagent/atom :default)
-        reveal-action      (fn [_]
-                             (reset! state :show)
-                             (when on-reveal
-                               (on-reveal)))
-        clear-action       #(reset! state :empty)
-        derive-action      #(js/alert "To be implemented")
-        choose-action      #(reset! state :show)
-        path-value         (reagent/atom (utils/get-formatted-derivation-path 3))
+        input-focused?     (reagent/atom false)
+        number-of-accounts (count (rf/sub [:wallet/accounts-without-watched-accounts]))
+        path-value         (reagent/atom (utils/get-formatted-derivation-path number-of-accounts))
         handle-path-change (fn [v]
                              (reset! path-value v)
-                             (when (empty? v)
-                               (clear-action)))
+                             (when (empty? v)))
         reset-path-value   (fn [_]
                              (reset! path-value "")
-                             (clear-action)
                              (when on-reset
-                               (on-reset)))]
+                               (on-reset)))
+        input-ref          (reagent/atom nil)]
     (fn [{:keys [theme]}]
-      (let [{:keys [public-key]}   (rf/sub [:profile/profile])
-            primary-name           (first (rf/sub [:contacts/contact-two-names-by-identity
-                                                   public-key]))
-            profile-picture        (rf/sub [:profile/image])
-            show-path-format-sheet #(rf/dispatch [:show-bottom-sheet {:content path-format-sheet/view}])]
+      (let [{:keys [public-key address]} (rf/sub [:profile/profile])
+            {:keys [password]}           (rf/sub [:get-screen-params])
+            primary-name                 (first (rf/sub [:contacts/contact-two-names-by-identity
+                                                         public-key]))
+            profile-picture              (rf/sub [:profile/image])
+            show-path-format-sheet       #(rf/dispatch [:show-bottom-sheet
+                                                        {:content path-format-sheet/view}])
+            derivation-path              (rf/sub [:wallet/derivation-path])
+            state                        (rf/sub [:wallet/derivation-path-state])
+            navigate-back-handler        #(if @input-focused?
+                                            (do
+                                              (.blur ^js @input-ref)
+                                              true)
+                                            (rf/dispatch [:navigate-to :screen/wallet.create-account]))
```

Comment:
Handle pressing back button on Android when keyboard is open

Reply:
A quick question, what does the `true` returned mean?